tests/cockpit-tests

```diff
@@ -53,16 +48,41 @@ echo "Starting testing"
 iteration=0
 while true; do
     fail=0
-    find cockpit -name '*.py?' -delete
     git -C cockpit fetch origin
     git -C cockpit reset --hard origin/master
-    if [ "$iteration" -eq 0 ]; then
-        cockpit/test/vm-download --prune
-    else
-        cockpit/test/vm-download
-    fi
+
+    # TODO: Soon we'll move this to an images bot
     iteration=$(( (iteration+1) % 25 ))
-    if ! cockpit/test/github-task "$@"; then
-        sleep 60
+    if [ -x cockpit/test/vm-download ]; then
+        if [ "$iteration" -eq 0 ]; then
+            cockpit/test/vm-download --prune
+        fi
     fi
+
+    (
+        # If the tasks file is executable, run it
+        if [ -x cockpit/.tasks ]; then
+            cockpit/.tasks
+
+        # If it exists then cat it
+        elif [ -e cockpit/.tasks ]; then
+            cat cockpit/.tasks
+
+        # No tasks found
+        else
+            echo "echo 'No tasks found in cockpit repo' >&2; exit 1"
```

Comment:
We don't set pipefail, so this should work. But I wonder why this doesn't just exit cleanly with 0 in this case?